docs/index.rst

```diff
@@ -558,6 +568,13 @@ skipping messages that will never be read.
    .. figure:: images/under_consumption_after.png
       :alt: image
 
+8. Return to the Data Streams view, find the ``wikipedia-activity-monitor``,
+   click on ``View Details`` and then ``Topic partitions``.  From this view
+   you can see the consumption status of the various topic and partitions for the
+   KStreams Application.
```

Comment:
Let's formally call this `Kafka Streams` instead of `KStreams`.  (throughout entire docs/index.html)